WebUI/src/components/AddLLMDialog.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+    <div class="dialog-container z-10">
+        <div class="dialog-mask absolute left-0 top-0 w-full h-full bg-black/55 flex justify-center items-center">
+            <div class="py-10 px-20 w-500px flex flex-col items-center justify-center bg-gray-600 rounded-3xl gap-6 text-white"
+                :class="{ 'animate-scale-in': animate }">
+              <p v-html= "i18nState.REQUEST_LLM_MODEL_NAME"></p>
+              <textarea class="rounded-xl border border-color-spilter flex-auto w-full h-auto resize-none" rows="1"
+                        :placeholder="languages.COM_LLM_HF_PROMPT" v-model="modelRequest" @keydown="fastGenerate"></textarea>
```

Comment:
works, but v-model is according to https://vuejs.org/guide/components/v-model a bidirectional channel -> we could update the value displayed to the customer from app side. 

This seems to be the standard, so we can leave it like this I guess. I am confused that vue does not seem to offer a one directional channel out of the box (and easy to use like v-model)

Reply:
this is actually even needed, since we want to reset the content as soon as the window is closed etc.
So I propose to let it be 🎵